example/notify_inval_inode.c

```diff
@@ -287,14 +287,14 @@ static void* update_fs_loop(void *data) {
         if (!options.no_notify && lookup_cnt) {
             /* Only send notification if the kernel is aware of the inode */
 
-            /* Some errors (ENOENT, EBADFD, ENODEV) have to be accepted as the
+            /* Some errors (ENOENT, EBADF, ENODEV) have to be accepted as they
              * might come up during umount, when kernel side already releases
              * all inodes, but does not send FUSE_DESTROY yet.
              */
             int ret =
                 fuse_lowlevel_notify_inval_inode(se, FILE_INO, 0, 0);
             if ((ret != 0 && !is_stop) &&
-                 ret != -ENOENT && ret != -EBADFD && ret != -ENODEV) {
+                 ret != -ENOENT && ret != -EBADF && ret != -ENODEV) {
                 fprintf(stderr,
```

Comment:
I don't think that is going to work - issue is that linux sends EBADFD here. It took me quite a while to fix these random unit test failures here. 
Do you know if FreeBSD kernel supports notifications at all?

Reply:
Yeah, you are right, there is not much that sends EBADFD, I'm going to double check. Reproducer for the github issues was a sleep in the loop - going to try that in the morning. 
If this is indeed a type, I wonder why test does not fail anymore, hmm.

Reply:
Sorry, got sick last Friday. When I just wanted to look into it now, I run into a new issue (see https://github.com/libfuse/libfuse/issues/979), need to look into that first.

Reply:
User error on my side, forgot to give the mount point, but still no reason to segfault inside of libfuse.
Tested it now, yeah, you are absolutely right, the error is actually `EBADF`. Interesting how it always passed tests - I had added that check to avoid random ci failures.